pkg/kv/kvpb/api.proto

```diff
@@ -71,6 +71,29 @@ enum RoutingPolicy {
   NEAREST = 1;
 }
 
+// ConnectionClass specifies the preferred RPC connection class to use for
+// sending or retransmitting the BatchRequest.
+//
+// This enum is similar and related to rpc.ConnectionClass, but the semantics
+// differ. This enum only defines classes used for BatchRequest traffic, and
+// only defines preferences. There are more classes in rpc package that are not
+// relevant here, such as RaftClass.
+enum ConnectionClass {
+  // DEFAULT declares no preference.
+  //
+  // Note that this does not necessarily correspond to rpc.DefaultClass. The
+  // actual connection class is chosen based on various factors. For example,
+  // for some critical system ranges, requests use rpc.SystemClass instead.
+  DEFAULT = 0;
+  // RANGEFEED declares a preference for RangefeedClass.
+  //
+  // As an example, at the time of writing this is used by changefeed initial
+  // scan requests. The rangefeed traffic is potentially bulky, so it has its
+  // dedicated connection class. Changefeed is a major consumer of rangefeed,
```

Comment:
I don't know if we should over-emphasize changefeed here -- physical cluster replication would also be a big consumer for this (and btw, you shoud probably change that too).